### ISSUE TITLE
Add llgo for Go

### ISSUE THREAD

**Issue by mys721tx** [mattgodbolt/compiler-explorer#996]
Hello all,

[llgo](https://llvm.org/svn/llvm-project/llgo/trunk/README.TXT), or formally known as [go-llvm](https://github.com/go-llvm/llgo), is a Go frontend for LLVM. Currently, Compiler Explorer offers the Go compiler and gccgo for Go. By including llgo, we can compare the difference in assembly generated those backends.

I wonder would it be possible to include llgo as one of the compilers for Go.

**Comment by mattgodbolt** [comment]
Absolutely. Should you wish to try this yourself, we have [some documentation on how to add a compiler](https://github.com/mattgodbolt/compiler-explorer/blob/master/docs/AddingACompiler.md).

**Comment by mattgodbolt** [comment]
Closing — the original llgo (LLVM Go frontend) was removed from the LLVM project and hasn't been maintained since ~2017. If there's interest in the newer Go+ llgo project, a fresh request would be welcome.

*(Triaged with LLM assistance)*
